examples/ProjectExample/pipinstaller/install_Functions.py

```diff
@@ -25,76 +23,83 @@ def checkPip():
         return False
     return True #Returns if whether it is installed or not
 
-#######################################
-#Function to check PIL install
-#######################################
+
 def checkPackage(pkg):
+    """Function to check PIL install.
+
+    pkg - a list of packages to check for
+    returns boolean annotating if package is installed or not
+    """
     try:
         _ = __import__(pkg[0])
-    except ImportError: #If the PIL package is not installed, this import will raise an ImportError
+    except ImportError: # If the package in question is not installed, this import will raise an ImportError
         packagesMissing.append(pkg)
         return False
-    print(pkg[0],"was found on the system")
-    return True #Returns if whether it is installed or not
+    print(pkg[0], "was found on the system")
+    return True # Returns if whether it is installed or not
```

Comment:
Small thing, could you quickly change it to "Returns whether it ...", just remove the if. I think I had something there before, doesn't make sense anymore.